databases/Assignment2_answers.sql

```diff
@@ -0,0 +1,316 @@
+1. Which laptop models are shown in the Product relation?
+SELECT DISTINCT(Model)
+FROM Product
+WHERE Type = laptop;
```

Comment:
You forgot to quote a lot in the questions below, but I'll avoiding pointing that out each time